### DISCUSSION TITLE
s/raja_yarcvuubdwgy-aihg

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#10] [Announcements]
# s/raja_yarcvuubdwgy-aihg

我从工作了十年的公司离职了

https://blog.ops-coffee.cn/s/raja_yarcvuubdwgy-aihg

**Comment by justinscript** [comment]
现在在哪个公司呀?

**Comment by yxy961112** [comment]
每次看都有不一样的感受

**Reply by ops-coffee** [reply]
我也是

**Comment by ztlong** [comment]
你现在应该后悔了

**Reply by ops-coffee** [reply]
后悔并没有，这两年也基本按照我之前的想法在做，还是有很多产出的，可以翻翻这两年的文章看看

**Comment by cmossss** [comment]
公司趋于平稳了就不再发展技术这点很认同，刚工作一年我就认识到现在的公司就是这样，已经不怎么注重技术研发了，早期员工或者高层自然是可以拿很多，但是对于我这种新兵蛋子无异于温水煮青蛙，看来自己也该考虑考虑以后了
